Assets/Scripts/EnemyShooting4Guns.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyShooting4Guns : MonoBehaviour
+{
+
+    [SerializeField] private float shootSpeed;
+    [SerializeField] private float shootTimer;
+    
+    [SerializeField] private List<Transform> bulletSpawns;
+    
+    [SerializeField] private GameObject bullet;
+    
+    private bool isShooting;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        isShooting = false;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (!isShooting)
+        {
+            StartCoroutine(Shoot());
+        }
+    }
+
+    IEnumerator Shoot()
+    {
+        isShooting = true;
+        GameObject newBullet1 = Instantiate(bullet, bulletSpawns[0].position, Quaternion.identity);
+        GameObject newBullet2 = Instantiate(bullet, bulletSpawns[1].position, Quaternion.identity);
+        GameObject newBullet3 = Instantiate(bullet, bulletSpawns[1].position, Quaternion.identity);
+        GameObject newBullet4 = Instantiate(bullet, bulletSpawns[1].position, Quaternion.identity);
```

Comment:
```suggestion
        GameObject newBullet1 = Instantiate(bullet, bulletSpawns[0].position, Quaternion.identity);
        GameObject newBullet2 = Instantiate(bullet, bulletSpawns[1].position, Quaternion.identity);
        GameObject newBullet3 = Instantiate(bullet, bulletSpawns[2].position, Quaternion.identity);
        GameObject newBullet4 = Instantiate(bullet, bulletSpawns[3].position, Quaternion.identity);
```

De andere 3 zitten dus op de zelfde spwan? Zijn er niet 4 dan.